lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -2255,10 +2255,13 @@ def update_service(self, name, old_service, new_service):
             name=name,
             **service_data
         )
```

Comment:
Shouldn't we also report warnings when updating?

Reply:
See the comment in the code. docker-py f***ed this up by not passing on the result which contains the warnings, so there's no way we can print them without modifying docker-py or talking directly to the docker daemon (without docker-py).